CONTRIBUTING.md

```diff
@@ -1,8 +1,79 @@
-# How to contribute
+# How to Contribute
 
-You'll need a copy of `pixi` and `git` in your machine. Then:
+## Try it locally
```

Comment:
```suggestion
```

I think we can remove this second header and remove "one level" from the subsequent headers so they are all `##` instead of `###`.